assets/js/base/components/shipping-calculator/address.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { __ } from '@wordpress/i18n';
+import Button from '@woocommerce/base-components/button';
+import { ShippingCountryInput } from '@woocommerce/base-components/country-input';
+import { ShippingCountyInput } from '@woocommerce/base-components/county-input';
+import TextInput from '@woocommerce/base-components/text-input';
+import { useState } from '@wordpress/element';
+import isShallowEqual from '@wordpress/is-shallow-equal';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+const ShippingCalculatorAddress = ( { address: initialAddress, onUpdate } ) => {
+	const [ address, setAddress ] = useState( initialAddress );
+
+	return (
+		<div className="wc-block-shipping-calculator-address">
+			<ShippingCountryInput
+				className="wc-block-shipping-calculator-address__input"
+				label={ __(
+					'Country / Region',
+					'woo-gutenberg-products-block'
+				) }
+				value={ address.country }
+				onChange={ ( newValue ) =>
+					setAddress( {
+						...address,
+						country: newValue,
+						county: '',
```

Comment:
I'd strongly suggest using `state` behind the scenes for this field. `country` and `county` are way too similar, and core also uses `state`.

Reply:
:100: Done in 576f46e94897abb68c0adacf895ddda9ce40b766.